aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -1093,113 +1105,113 @@ resource "aws_elasticache_cluster" "bar" {
-  tags = {
+resource "aws_subnet" "test" {
+    vpc_id            = "${aws_vpc.test.id}"
+    cidr_block        = "192.168.0.0/20"
+    availability_zone = "${data.aws_availability_zones.test.names[0]}"
+    tags = {
         Name = "tf-acc-elasticache-cluster-in-vpc"
     }
 }
 
-resource "aws_elasticache_subnet_group" "bar" {
-    name = "tf-test-cache-subnet-%03d"
+resource "aws_elasticache_subnet_group" "test" {
+    name        = "tf-test-cache-subnet-%03d"
     description = "tf-test-cache-subnet-group-descr"
-    subnet_ids = ["${aws_subnet.foo.id}"]
+    subnet_ids  = ["${aws_subnet.test.id}"]
 }
 
-resource "aws_security_group" "bar" {
-    name = "tf-test-security-group-%03d"
+resource "aws_security_group" "test" {
+    name        = "tf-test-security-group-%03d"
     description = "tf-test-security-group-descr"
-    vpc_id = "${aws_vpc.foo.id}"
+    vpc_id      = "${aws_vpc.test.id}"
     ingress {
-        from_port = -1
-        to_port = -1
-        protocol = "icmp"
+        from_port   = -1
+        to_port     = -1
+        protocol    = "icmp"
         cidr_blocks = ["0.0.0.0/0"]
     }
 }
 
-resource "aws_elasticache_cluster" "bar" {
+resource "aws_elasticache_cluster" "test" {
     // Including uppercase letters in this name to ensure
     // that we correctly handle the fact that the API
     // normalizes names to lowercase.
-    cluster_id = "tf-%s"
-    node_type = "cache.m1.small"
-    num_cache_nodes = 1
-    engine = "redis"
-    engine_version = "2.8.19"
-    port = 6379
-    subnet_group_name = "${aws_elasticache_subnet_group.bar.name}"
-    security_group_ids = ["${aws_security_group.bar.id}"]
-    parameter_group_name = "default.redis2.8"
-    notification_topic_arn      = "${aws_sns_topic.topic_example.arn}"
-    availability_zone = "${data.aws_availability_zones.available.names[0]}"
-}
-
-resource "aws_sns_topic" "topic_example" {
-  name = "tf-ecache-cluster-test"
+    cluster_id             = "tf-%s"
+    node_type              = "cache.m1.small"
+    num_cache_nodes        = 1
+    engine                 = "redis"
+    engine_version         = "2.8.19"
+    port                   = 6379
+    subnet_group_name      = "${aws_elasticache_subnet_group.test.name}"
+    security_group_ids     = ["${aws_security_group.test.id}"]
+    parameter_group_name   = "default.redis2.8"
+    notification_topic_arn = "${aws_sns_topic.test.arn}"
+    availability_zone      = "${data.aws_availability_zones.test.names[0]}"
+}
+
+resource "aws_sns_topic" "test" {
+    name = "tf-ecache-cluster-test"
 }
 `, acctest.RandInt(), acctest.RandInt(), acctest.RandString(10))
 
 var testAccAWSElasticacheClusterMultiAZInVPCConfig = fmt.Sprintf(`
-data "aws_availability_zones" "available" {
-  # InsufficientCacheClusterCapacity: cache.m1.small (VPC) is not currently supported in the availability zone us-east-1b
-  blacklisted_zone_ids = ["use1-az1"]
-  state                = "available"
+data "aws_availability_zones" "test" {
```

Comment:
Should remain as "available" -- see comment below

Reply:
👍